src/data/teams.js

```diff
@@ -6,7 +6,7 @@ import icpc from "../img/icpc.svg";
 import design from "../img/design.svg";
 import ai from "../img/ai.svg";
 import cyber from "../img/cyber.svg";
-// import studio from "../img/studio.svg";
+import studio from "../img/studio.svg";
```

Comment:
Let's change the studio icon from the current wordmark style to a simple logo (without the words). You can find the logo [here](https://drive.google.com/file/d/19Z6OyTsVvYGYfQmO23NsoKZXW9zi60ax/view?usp=sharing).

Reply:
For this logo, you're referring to the logo import in resources.js and not teams right, since the logo is wordmarked for every committee in teams.js?

Reply:
Oops, yes my mistake. Commented on the wrong file.